cypress/e2e/po/pages/explorer/projects-namespaces.po.ts

```diff
@@ -29,7 +30,63 @@ export default class ProjectNamespacePagePo extends PagePo {
     return this.createProjectNamespaceButton().click();
   }
 
+  createProjectButton() {
+    return this.self().getId('masthead-create');
+  }
+
+  createProjectButtonClick(): Cypress.Chainable {
+    return this.createProjectButton().click();
```

Comment:
given above there needs to be a `createForm` fn in that header po and called from here

```
  createForm() {
    return this.self().find('[data-testid="masthead-create"]').click();
  }
```

Not quite sure on `createForm` name, but you get the drift